src/extension/importer/core/src/test/java/org/geoserver/importer/DataFormatTest.java

```diff
@@ -16,8 +16,8 @@
 public class DataFormatTest extends ImporterTestSupport {
 
     @Test
-    public void testLookupShapefile() {
-        DataFormat format = DataFormat.lookup(new File("foo.shp"));
+    public void testLookupShapefile() throws Exception {
+        DataFormat format = DataFormat.lookup(new File(tmpDir(), "foo.shp"));
```

Comment:
Thanks @NielsCharlier, I noticed this cruft and am pleased that you have removed it. I can confirm that this commit fixes the importer and gwc cruft.